content/SCALE/Storage/Pools/EncryptionSCALE.md

```diff
@@ -41,85 +40,207 @@ These algorithms provide authenticated encryption with block ciphers.
 
-Look at the *Encryption Options* and, when the parent dataset is unencrypted, uncheck *Inherit (non-encrypted)*.
+Using the passphrase option might be easier to manage but choose a complex phrase not easy to guess.
+
+{{< hint danger>}}
+Keep both encryption keys and/or passphrases safeguarded in a secure and protected place. 
+Losing encryption keys or passphrases can result in permanent data loss!
+{{< /hint >}}
+
+{{< /tab >}}
+{{< /tabs >}}
 
-![EncryptionOptionsSCALE](/images/SCALE/EncryptionOptionsSCALE.png "Dataset Encryption Options")
+After configuring the new dataset encryption settings, you can change any or leave the remaining settings the same as the parent dataset.
+Click **Save**. 
+The new dataset displays on the **Storage** screen below its parent dataset. 
+If you encrypt a dataset, an unlocked icon displays to the right of its name and a locked icon displays to the right of the root dataset name.
+It remains unlocked until you lock it.
 
-Next, choose which *Encryption Type* of authentication to use: a *Key* or a *Passphrase*.
-The remaining options are the same as a new pool.
-Encrypted datasets show additional icons in the **Storage** list.
+![StorageDatasetList](/images/SCALE/StorageDatasetList.png "Storage Screen Dataset List")
+
+### Changing Dataset Encryption
+
+You can change dataset encryption settings using **Encryption Options** on the **Dataset Action** menu. 
+The encryption options only display for datasets with encryption configured. 
+The **Edit Encryption Options *datasetfullpath*** configuration window displays. 
+The window name includes the selected dataset's full path name. 
+In this example it has the root or parent dataset, the child dataset without encryption, and the selected child-of-the-child dataset with encryption configured (i.e., *tank/tank-child/tank-child-encrypt*).
+
+![EditEncryptionOptionsWindow](/images/SCALE/EditEncryptionOptionsWindow.png "Edit Encryption Options")
+
+After making any changes, click the **Confirm** checkbox to check-mark it and then click **Save**. 
+
+{{< hint danger >}}
+Save any change to the encryption key or passphrase, update your saved passcodes and keys file, and back it up. 
+{{< /hint >}} 
 
 ### Locking and Unlocking Datasets
 
 TrueNAS displays a dataset's status with an icon:
 
-* Dataset unlocked icon: <i class="material-icons" aria-hidden="true" title="<unlocked>">lock_open</i>
+* Dataset unlocked icon: <span class="iconify" data-icon="mdi:lock-open-variant"></span>
 * Dataset locked icon: <i class="material-icons" aria-hidden="true" title="<locked>">lock</i>
 
-Encrypted datasets can only be locked and unlocked if they use a passphrase instead of a key.
-Before locking a dataset, verify that it is not currently in use, then click <i class="fa fa-ellipsis-v" aria-hidden="true" title="Options"></i>&nbsp; (Dataset Actions) and *Lock*.
+You use a passphrase instead of a key to lock or unlock encrypted datasets.
```

Comment:
replace with:
You can only lock and unlock encrypted datasets when it is secured with a passphrase instead of a key file.